m4_sessions/1706-inning/group_six.md

```diff
@@ -1,41 +1,56 @@
 
 ### Session Outcomes: 
--Reaffirming that we're in the right place: Turing is hard and we all have disparate backgrounds, but we're on the right track for us
--Insight into the direction of coding/bootcamps: how did we get here and where is this trend going?
--Be able to identify efficiencies and defficiencies of coding school alums vs CS degree holders
-
-### Session Outline:
--give SHORT presentation
-  -topics: code school trend, benefits/downsides of code school, is code school the next blue collar economy? Is the junior dev market saturated, how are we better prepared than CS degree holders
--break into groups of 4 or 5, each gear-up leader will be in a group
--each group discuss the same topic for ~7 minutes at a time
--discuss severaldifferent topics, mixing up the groups
-
-### Names of group members:
-Jack Bevis
-Devin Dixon
-Keji Amos
-Chris Brandrowsky
-Anders Wood
-
-##### Notes
-benefits/downsides to coding at code school:
-benefits/downsides to cgetting coding in highschool/college?
-
--The rise of coding/bootcamp schools
-  -history of this trend
-  -profit vs nonprofit
-  -spectrum of involvement: weekends vs 7 months
-  -
--What is the dynamic of CS majors vs. code school alums
-  -how do curricula differ with these two approaches?
-  -how are students of these two schools prepared for jobs differently?
-  -why do we choose to come here?
-  -
--is junior dev hiring market saturate?
--is coding the next blue colar job?
--
+* Affirm that we're in the right place: Turing is hard and we all have disparate backgrounds, but we're on the right track for us
+* Provide insight into the direction of coding/bootcamps broadly: how did we get here and where is this trend going?
+* Identify efficiencies and deficiencies of coding school alums vs CS degree holders
+
+### Session Outline
+* Present for 2 minutes on a topic
+* Break students into 6 groups
+* Groups discuss three prompts for the topic for 7 minutes total
+* After 7 minutes, 2 groups join (3 groups) and share interesting/controversial/exciting points that came up in their group discussion
+* Repeat for remaining 2 topics
+* After discussing all the topics, rejoin as one big group for the last 5 minutes to discuss most interesting/controversial topics 
+
+##### TOPIC 1 (Jack, Devin): What was our process for arriving at Turing? (CS degree vs boot camp, 12 week course, online courses, advanced degrees)
+2 Minute Presentation Notes:
+* Jack: quick recap of the decision making process I made that led me to Turing
+* Dev: same personal style recap
+
+Prompts:
+* Why did you choose Turing? What influenced your decision?
```

Comment:
I'd have these prompts for each section on the board or on a slide so that people can reference it.  Many times people forget the questions if they are not visible.